skipruntime-ts/server/src/rest.ts

```diff
@@ -16,7 +16,8 @@ export function controlService(service: ServiceInstance): express.Express {
     try {
       const uuid = crypto.randomUUID();
       service.instantiateResource(uuid, req.params.resource, req.body as Json);
-      res.status(201).send(uuid);
+      const prefix = process.env["SKIP_RESOURCE_PREFIX"];
+      res.status(201).send(prefix ? `${prefix}/${uuid}` : uuid);
```

Comment:
We might as well get rid of the `/` and leave full control of the prefix to the user, what do you think?

Reply:
That's more flexibility for the user to be sure, but this makes the url-splitting logic in the reverse proxy dead simple. I guess I'd be slightly worried about people forgetting a delimiter or choosing a bad one and otherwise messing that up.